### DISCUSSION TITLE
Permission error for rowy workspace

### DISCUSSION THREAD

**Discussion by thezyrie** [AntlerVC/firetable#1417] [Support / Q&A]
I can log in and see my dashboard but whenever I click on my table - for which I created and have every role -  I get `You do not have access to this project. Please contact the project owner.` But I am the project owner and it's adding a .rowy to my email which might be why its failing
![image](https://github.com/rowyio/rowy/assets/20506437/36408667-23ec-4785-99b6-aece98e27f8d)

**Comment by harinij** [comment]
Hi @thezyrie. You can try a couple of things here:
1) check if its a cache issue, clear cache and refresh and try again
2) check if you do not have owner perms on your firebase project
